scenes/game/local_game.gd

```diff
@@ -11144,6 +11158,19 @@ func do_quit(player_id : Enums.PlayerId, reason : Enums.GameOverReason):
 	event_queue += events
 	return true
 
+func do_timeout(player_id : Enums.PlayerId, reason : Enums.GameOverReason):
```

Comment:
name of this needs to be the clock thing
And does this need a game over reason passed in? Probably not?